airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -99,8 +101,13 @@ def read_records(self, **kwargs) -> Iterable[Mapping[str, Any]]:
 class BulkSalesforceStream(SalesforceStream):
 
     page_size = 30000
-    JOB_WAIT_TIMEOUT_MINS = 10
-    CHECK_INTERVAL_SECONDS = 2
+    DEFAULT_WAIT_TIMEOUT_MINS = 10
+    MAX_CHECK_INTERVAL_SECONDS = 2.0
+    MAX_RETRY_NUMBER = 3
+
+    def __init__(self, wait_timeout: Optional[int], **kwargs):
```

Comment:
every time we pass a time duration we should either use a data type which encodes the unit of time or we should embed the unit of time in the var name e.g: `wait_timeout_in_minutes`